build.gradle

```diff
@@ -0,0 +1,110 @@
+allprojects  {
+    apply plugin: 'maven'
+
+    group = 'microprofile.gift.demo'
+    version = '0.0.1-SNAPSHOT'
+}
+
+description = 'Acme gift giving using microservices'
+
+subprojects {
+    apply plugin: 'java'
+    sourceCompatibility = 1.8
+    targetCompatibility = 1.8
+    tasks.withType(JavaCompile) {
+        options.encoding = 'UTF-8'
+    }
+
+    repositories {
+        mavenLocal()
+
+        maven { url "https://oss.sonatype.org/content/repositories/netwasdev-1087" }
```

Comment:
Remove this bad URL. It looks like a staging release.

Reply:
done